target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -142,6 +142,10 @@ pcs,cr3000)
 qihoo,c301)
 	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	;;
+teltonika,rut230)
+       ucidef_set_led_switch "lan" "LAN" "rut230:green:lan" "eth0"
+       ucidef_set_led_switch "wan" "WAN" "rut230:green:wan" "eth1"
```

Comment:
This tells me you have not checked LEDs, as this won't work. You are mixing switch and netdev syntax here.